### PR TITLE
control-group: abstract helper for grouping different controls

### LINKED ISSUE #540: control-group: abstract helper for grouping different controls
1) Often we need to group different controls:
input

2) For now blocks like `checkbox-group` and `radio-group` are almost totaly identical and should be rafactored:

### PR DESCRIPTION
**Pull request by mishanga**
close #540

### CODE REVIEW

**Comment by mishanga** [comment]
@gela-d @dfilatov

File: design/common.blocks/button/_theme/button_theme_normal.roo [hunk 1]
```diff
@@ -1,4 +1,5 @@
-.button_theme_normal
+$ctx = '.button_theme_normal';
```
Comment (narqo) [inline design/common.blocks/button/_theme/button_theme_normal.roo:1]: Wouldn't this variable become global for all blocks during the build process?

**Comment by dfilatov** [comment]
I haven't seen any problems except hellish css selectors )

**Comment by mishanga** [comment]
@dfilatov I was trying to call Satan by those selectors ;)

File: design/common.blocks/button/_theme/button_theme_normal.roo (hunk 1 above)
Comment (mishanga) [inline design/common.blocks/button/_theme/button_theme_normal.roo:1]: Yep, it would. Is it problem?

Comment (narqo) [inline design/common.blocks/button/_theme/button_theme_normal.roo:1]: My initial warn was about potential name collisions, but I think this is not a problem, until we don't use it in other files

**Comment by dfilatov** [comment]
@mishanga Has it done?

**Comment by mishanga** [comment]
Not yet, I'll finish it soon.

**Comment by dfilatov** [comment]
LGTM

**Pull request merged**